tests/functional/plugincheck.js

```diff
@@ -0,0 +1,91 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global casper */
+'use strict';
+
+var config = require('../lib/config');
+var helpers = require('../lib/helpers');
+var path = '/plugincheck/';
+var url = config.base() + path;
+
+casper.options.pageSettings.loadPlugins = true;
+
+
+casper.test.begin('PluginCheck, Elements' + url, 4, function suite(test) {
+
+    casper.start();
+
+    casper.userAgent(config.userAgent.fx);
+
+    casper.thenOpen(url, function() {
+        test.assertHttpStatus(200);
+        test.assertResourceExists(/media\/img\/plugincheck\/screenshots\/addons-screenshot(\.\w{1,}){0,}\.jpg/);
```

Comment:
Could this be simplified to just `test.assertResourceExists(/media\/img\/plugincheck\/screenshots\/addons-screenshot/);` maybe - thoughts?